### ISSUE TITLE
Improve code coverage accuracy

### ISSUE THREAD

**Issue by bigmstone** [StackStorm/st2#3227]
From #3223 - Codecov comments have been disabled since they're mostly noise until we can accuratly represent code coverage.

**Comment by Kami** [comment]
Yeah this shouldn't be a blocker for v2.2.1, especially since it's not tied to a specific StackStorm release and we can work on it at any time :)
